### PR TITLE
MDS-3200 Add Core uploaded NoW data to permit table.

### PR DESCRIPTION
**Pull request by JustinMacaulay**
Pulls the NoW uploaded Map documents and the final application package to be shown on the permit table.

### CODE REVIEW

File: services/core-web/src/components/noticeOfWork/applications/process/ProcessPermit.js [hunk 1]
```diff
@@ -156,11 +175,71 @@ export class ProcessPermit extends Component {
...
+    return noticeOfWork.documents
+      .filter((document) => document.is_final_package)
+      .map((document) => ({
+        document_name: document.mine_document.document_name,
+        document_upload_date: formatDate(document.mine_document.upload_date),
+      }));
+  };
+
+  afterSuccess = (values, message, code) => {
+    console.log(values);
```
Comment (mmumby) [inline services/core-web/src/components/noticeOfWork/applications/process/ProcessPermit.js:242]: remove console.log()

File: services/core-web/src/components/noticeOfWork/applications/process/ProcessPermit.js [hunk 2]
```diff
@@ -214,7 +334,15 @@ export class ProcessPermit extends Component {
...
       payload,
-      "Successfully created document and attached it to Notice of Work"
+      "Successfully created document and attached it to Notice of Work",
+      () => {
+        if (
+          documentType.now_application_document_type_code === "PMA" ||
+          documentType.now_application_document_type_code === "PMT"
+        ) {
+          afterSuccess(values, "Permit has been successfully issued for this application.", "AIA");
+        }
```
Comment (mmumby) [inline services/core-web/src/components/noticeOfWork/applications/process/ProcessPermit.js:344]: can you use the const approvedCode for every instance of "AIA"

**Review by exzizt — COMMENTED** [review]
Great work. I made a commit to return some promises that will be needed for redux forms to properly set the submitting prop and prevent the spamability of buttons.

Happy to approve once Meredith's comments are addressed.

File: services/core-web/src/components/noticeOfWork/applications/process/ProcessPermit.js (hunk 2 above)
Comment (JustinMacaulay) [reply services/core-web/src/components/noticeOfWork/applications/process/ProcessPermit.js:343]: Done

File: services/core-web/src/components/noticeOfWork/applications/process/ProcessPermit.js (hunk 1 above)
Comment (JustinMacaulay) [reply services/core-web/src/components/noticeOfWork/applications/process/ProcessPermit.js]: Done

**Comment by mmumby** [comment]
> Great work. I made a commit to return some promises that will be needed for redux forms to properly set the submitting prop and prevent the spamability of buttons.
> 
> Happy to approve once Meredith's comments are addressed.

My comments are addressed! is this ready to go now?

**Review by mhuseinov — APPROVED** [review]

**Pull request merged**